frame/babe/src/lib.rs

```diff
@@ -835,3 +859,42 @@ fn compute_randomness(
 
 	sp_io::hashing::blake2_256(&s)
 }
+
+pub mod migrations {
+	use super::*;
+	use frame_support::pallet_prelude::{ValueQuery, StorageValue};
+
+	struct __OldNextEpochConfig;
+	impl frame_support::traits::StorageInstance for __OldNextEpochConfig {
+		fn pallet_prefix() -> &'static str { "BabeApi" }
```

Comment:
This is wrong.

Currently this is `Babe` and not `BabeApi`. 

Also this should probably be configurable.

Reply:
I've made it configurable, but I think it still is `BabeApi`. See https://github.com/paritytech/polkadot/pull/2467#discussion_r590074182.